frontend/src/pages/lesson/InsightsDuring.jsx

```diff
@@ -221,24 +222,10 @@ const InsightsDuring = (props) => {
       <Grid container spacing={3} className={classes.widget}>
         <PaperWithHeader
           headercolor={theme.widget.secondary.main}
-          headertitle="Topcategorieën"
+          headertitle="Correctheid categorieën"
         >
           <Paper className={classes.paper}>
-            {categories.slice(0, 3).map((category, index) => (
-              <ProgressBar
-                key={category.key}
-                title={`${index + 1}. ${convertCategoryToString(category.key)}`}
-                value={category.correctness}
-              />
-            ))}
-          </Paper>
-        </PaperWithHeader>
-        <PaperWithHeader
-          headercolor={theme.widget.secondary.main}
-          headertitle="Probleemcategorieën"
-        >
-          <Paper className={classes.paper}>
-            {categories.slice(-3).map((category, index) => (
+            {categories.map((category, index) => (
```

Comment:
Laten we niet meer alleen de top 3 zien? Anders wordt het een heel lang component als je veel soorten vragen hebt. Misschien een popup laten zien om alles te tonen?

Reply:
kan ook alles laten zien, maar een maxHeight zetten, en een overflow-y: scroll